### PR TITLE
Add nm version property getter

### PR DESCRIPTION
**Pull request by rhrazdil**
Add getter for NetworkManager version property 

Signed-off-by: Radim Hrazdil <rhrazdil@redhat.com>

### CODE REVIEW

File: src/property.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package networkmanager
```
Comment (phoracek) [inline src/property.go]: Let's just name it version.go? Many of the modules work with properties.

File: examples/get-version.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+  "fmt"
+
+  networkmanager "github.com/phoracek/networkmanager-go/src"
```
Comment (phoracek) [inline examples/get-version.go]: I think that this module needs to go through gofmt.

**Review by phoracek — CHANGES_REQUESTED** [review]

File: src/property.go (hunk 1 above)
Comment (rhrazdil) [reply src/property.go]: Done

File: examples/get-version.go (hunk 2 above)
Comment (rhrazdil) [reply examples/get-version.go]: Fixed

**Review by phoracek — APPROVED** [review]
Thanks, looks great!

```
[networkmanager-go] go run examples/get-version.go
1.26.8
```

**Pull request merged**